hub/management/commands/import_council_mrp_data.py

```diff
@@ -0,0 +1,159 @@
+from mysoc_dataset import get_dataset_df
+
+from hub.import_utils import add_gss_codes, filter_authority_type
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+
+class Command(MultipleAreaTypesMixin, BaseImportFromDataFrameCommand):
+    message = "Importing council MRP data"
+    uses_gss = True
+    do_not_convert = True
+    cons_row = "gss_code"
+
+    area_types = ["STC", "DIS"]
+
+    defaults = {
+        "data_type": "integer",
+        "category": "opinion",
+        "source_label": "Public First MRP Polling, commissioned by Onward UK, adjusted for the demographics of the local area by mySociety.",
+        "release_date": "July 2022",
+        "source": "https://www.publicfirst.co.uk/new-public-first-polling-for-onward.html",
+        "source_type": "google sheet",
+        "subcategory": "net_zero_support",
+        "table": "areadata",
+        "default_value": 10,
+        "data_url": "",
+        "exclude_countries": ["Northern Ireland"],
+        "comparators": DataSet.numerical_comparators(),
+        "unit_type": "percentage",
+        "unit_distribution": "people_in_area",
+    }
+
+    survation_defaults = {
+        "data_type": "percent",
+        "category": "opinion",
+        "source_label": "Survation MRP polling, commissioned by RenewableUK, adjusted for the demographics of the local area by mySociety.",
+        "release_date": "September 2022",
+        "source": "https://www.renewableuk.com/news/615931/Polling-in-every-constituency-in-Britain-shows-strong-support-for-wind-farms-to-drive-down-bills.htm",
+        "source_type": "google sheet",
+        "table": "areadata",
+        "exclude_countries": ["Northern Ireland"],
+        "default_value": 50,
+        "comparators": DataSet.numerical_comparators(),
+        "unit_type": "percentage",
+        "unit_distribution": "people_in_area",
+    }
+
+    data_sets = {
+        "council_nz_support": {
+            "defaults": {
+                **defaults,
+                "label": "Support Net Zero",
+            },
+            "col": "Q02_Support",
+        },
+        "council_nz_neutral": {
+            "defaults": {
+                **defaults,
+                "label": "Neither Support nor Oppose Net Zero",
```

Comment:
Sentence case for dataset `label`s. (With the added complication that “Net Zero” tends to be treated as a proper noun, because it’s a sort of special concept.)

So this label would be “Neither support nor oppose Net Zero”.